sql/parser/ast.go

```diff
@@ -37,20 +38,22 @@ type Statement interface {
 	statement()
 }
 
-func (*Union) statement()  {}
-func (*Select) statement() {}
-func (*Insert) statement() {}
-func (*Update) statement() {}
-func (*Delete) statement() {}
-func (*Set) statement()    {}
-func (*Use) statement()    {}
-func (*DDL) statement()    {}
+func (*Union) statement()          {}
+func (*Select) statement()         {}
+func (*Insert) statement()         {}
+func (*Update) statement()         {}
+func (*Delete) statement()         {}
+func (*Set) statement()            {}
+func (*Use) statement()            {}
+func (*CreateDatabase) statement() {}
+func (*CreateIndex) statement()    {}
+func (*CreateTable) statement()    {}
+func (*DDL) statement()            {}
```

Comment:
I believe these are in the order that they appear in the file. I'd prefer not to edit the order right now, though my OCD tends to agree with you.